test/e2e/apps/disruption.go

```diff
@@ -62,6 +62,7 @@ var defaultLabels = map[string]string{"foo": "bar"}
 
 var _ = SIGDescribe("DisruptionController", func() {
 	f := framework.NewDefaultFramework("disruption")
+	f.NamespacePodSecurityEnforceLevel = "privileged"
```

Comment:
suggest making this typed (not a string) and using the constants in https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/pod-security-admission/api/constants.go

Reply:
ack, I wanted to reduce inter-package dependencies, but can reference the constants :+1:

Reply:
fixed